fieldmanager.php

```diff
@@ -243,12 +243,15 @@ function _fieldmanager_registry( $var, $val = null ) {
  *
  * @see fm_calculate_context() for detail about the returned array values.
  *
+ * @param bool $force Optional. If true, FM will recalculate the current context.
+ *                    This is necessary for testing and perhaps other
+ *                    programmatic purposes.
  * @return array Contextual information for the current request.
  */
-function fm_get_context() {
+function fm_get_context( $force = false ) {
```

Comment:
To me, `$force` reads as if you're forcing FM to return something when it otherwise wouldn't, which isn't quite what's happening, if I'm following correctly. Maybe FM could do with a dedicated `clear` or `recalculate` workflow, or a way to get a new calculated context independently of the static one? (I realize this comment is coming a little late in the 1.0 cycle, though.)